UPGRADING.md

```diff
@@ -6,6 +6,12 @@ Upgrade from 4.X to 5.X
 
 This release contains **breaking changes**.
 
+```groovy
+dependencies {
+    classpath 'com.bugsnag:bugsnag-android-gradle-plugin:5.+'
```

Comment:
Please don't recommend wildcard dependencies, as they result in reproducibility

Reply:
I take your point and it wasn't intended as a recommendation as such.  How would you suggest I express the dependency?  Perhaps just as pseudo code like `5.<latest>`?

Reply:
Hmmm. Maybe something like `5.x.x` to make it clear that it's just a placeholder?